app/networkapi/people/serializers.py

```diff
@@ -24,7 +9,13 @@ class PersonSerializer(serializers.ModelSerializer):
     """
     affiliations = serializers.StringRelatedField(many=True)
     internet_health_issues = serializers.StringRelatedField(many=True)
-    links = LinkSerializer(many=True)
+    links = serializers.SerializerMethodField()
+
+    def get_links(self, instance):
+        return {
+            'twitter': instance.twitter_url,
+            'linkedIn': instance.linkedin_url,
```

Comment:
lets keep it consistent across all the attributes and have the key as `linkedin`